.travis.yml

```diff
@@ -6,37 +8,25 @@ dist: trusty
 services:
   - docker
 
-matrix:
-  fast_finish: true
-  allow_failures:
-    - go: master
-
 go:
   - 1.8
   - 1.9
-  - master
 
 before_install:
-  - pyenv versions
-  - pyenv global 3.5
-  - /usr/bin/env python3 --version
-  - sudo apt-get install -y netcat softhsm
-  - go get github.com/wadey/gocovmerge
+  - sudo apt-get update
+  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
```

Comment:
This seems weird, but /shrug travis?

Reply:
Need to have up-to-date docker-ce version to make builds args work in $FROM. Otherwise we can only build with one version of Go.